File: test/fixtures/elixir/post_basic_auth_url_encoded_data.ex
response = HTTPoison.post!(
  "http://localhost:28139/api/oauth/token/",
  {:form, [
    {"grant_type", "client_credentials"}
  ]},
  [
    {"Content-Type", "application/x-www-form-urlencoded"}
  ],
  [hackney: [basic_auth: {"foo", "bar"}]]
)

File: test/fixtures/csharp/post_basic_auth_url_encoded_data.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, "http://localhost:28139/api/oauth/token/");

request.Headers.Add("Authorization", "Basic " + Convert.ToBase64String(System.Text.ASCIIEncoding.ASCII.GetBytes("foo:bar")));

request.Content = new StringContent("grant_type=client_credentials");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/x-www-form-urlencoded");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();
